src/dev/build/build_distributables.ts

```diff
@@ -142,42 +143,49 @@ export async function buildDistributables(log: ToolingLog, options: BuildOptions
 
     if (options.createDebPackage) {
       // control w/ --deb or --skip-os-packages
-      artifactTasks.push(Tasks.CreateDebPackage);
+      artifactTasks.push(Tasks.CreateDebPackageX64);
```

Comment:
wouldn't this mean, that we're always building both platforms (i.e.: `--all-platforms` even without adding this flag?)